robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java

```diff
@@ -74,4 +74,16 @@ public void HSVToColorShouldReverseColorToHSV() {
 
       assertThat(Color.HSVToColor(hsv)).isEqualTo(Color.RED);
   }
+
+  @Test
+  public void HSVToColorValueShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[]{0f, 0f, -1.0f})).isEqualTo(Color.BLACK);
+    assertThat(Color.HSVToColor(new float[]{0f, 0f, 2.0f})).isEqualTo(Color.WHITE);
+  }
+
+  @Test
+  public void HSVToColorSaturationShouldBePinned() {
+    assertThat(Color.HSVToColor(new float[]{0f, -1.0f, 0.5f})).isEqualTo(0xff808080);
```

Comment:
Maybe we also can add these tests to ctesque.

Reply:
@mlebedy if you like, you also can add these tests to ctesque module for Emulator testing.

Reply:
OK! Here is my PR for the integration test: https://github.com/robolectric/robolectric/pull/8281